packages/search/src/features/registration/birth/service.ts

```diff
@@ -302,10 +304,14 @@ function createDeclarationIndex(
       (code) => code.system === 'http://opencrvs.org/doc-types'
     )
 
-  body.informantType =
+  const otherInformantType =
     (contactPersonRelationshipExtention &&
       contactPersonRelationshipExtention.valueString) ||
     (contactPersonExtention && contactPersonExtention.valueString)
+
+  const informantType = getInformantType(bundle as ValidRecord)
+
+  body.informantType = informantType || otherInformantType
```

Comment:
we probably don't need the `otherInformantType` since we can get it all from the same resource of the bundle.

Reply:
@Nil20 in that case we should remove it 👍